code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1155,78 +1155,21 @@
 					I = overlays_cache[23]
 			damageoverlay.overlays += I
 
-		if( stat == DEAD )

```

Comment:
Believe it or not, all of this was duplicate code! Science, right?

(And yes, actually tested.)